tests/json.spec.js

```diff
@@ -108,10 +108,10 @@ describe('JSON', function() {
   it('*_ieeg.json sidecars should have required key/value pairs', function() {
     var jsonObj = {
       TaskName: 'Audiovis',
-      Manufacturer: 'TDT',
-      PowerLineFrequency: 50,
       SamplingFrequency: 10,
-      iEEGReference: 'reference',
+      PowerLineFrequency: 50,
+      SoftwareFilters: 'high-pass',
```

Comment:
@choldgraf I think the way the software filter is specified does not conform to what we require:
```json
    "SoftwareFilters": {
      "anyOf": [
        {"type": "object", "additionalProperties": {"type":"object"}},
        {"type": "string", "pattern": "^n/a$"}
      ]
    }
```

It should either be: a string `"na"`

... or it could be an object of objects: `{'HighPass': {'HalfAmplitudeCutOffHz': 1, 'RollOff: '6dB/Octave'}}`

```suggestion
      SoftwareFilters: {'HighPass': {'HalfAmplitudeCutOffHz': 1, 'RollOff: '6dB/Octave'}},
```